proxy_test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"flag"
+	"github.com/iann0036/iamlive/faillog"
+	"github.com/stretchr/testify/assert"
+	"net/http/httptest"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func TestSimpleRequest(t *testing.T) {
+	if testing.Short() {
+		t.Skipf("skipping e2e test")
+	}
+
+	parseConfig()
+	flag.Parse()
+	loadMaps()
+	readServiceFiles()
+	*modeFlag = "proxy"
```

Comment:
This is pretty ugly, but I figured I didn't want to make bigger changes in a single PR